netket/graph/lattice.py

```diff
@@ -241,6 +252,10 @@ def __init__(
             distance_atol: Distance below which spatial points are considered equal for
                 the purpose of identifying nearest neighbors.
             point_group: Default `PointGroup` object for constructing space groups
+            max_neighbor_order: For :code:`max_neighbor_order == k`, edges between up
+                to :math:`k`-nearest neighbor sites (measured by their Euclidean distance)
+                are included in the graph. The edges can be distiguished by their color,
+                which is set to :math:`k`.
```

Comment:
should be k-1, right?

Reply:
That's true, it's `k - 1`. While potentially confusing, I think having nearest-neighbors be zero has the benefit that the default (only nearest neighbors) is equivalent to an uncolored graph (which just has `color==0` for all edges in NetKet).